pynucastro/rates/rate.py

```diff
@@ -1027,6 +1033,94 @@ def plot(self, Tmin=1.e8, Tmax=1.6e9, rhoYmin=3.9e8, rhoYmax=2.e9,
         return fig
 
 
+class DerivedRate(Rate):
+    """
+    This class is a derived class from `Rate` with the purpose of computing the inverse rate
+    by the application of detailed balance to the forward reactions.
+    """
+
+    def __init__(self, rate, use_pf=False, use_A_nuc=False):
+
```

Comment:
this seems a little odd to me.  `DerivedRate` subclasses `Rate`, but the `__init__` requires a `Rate` object.  That doesn't seem very pythonic.

@yut23 thoughts?

Reply:
well.... maybe that's what I did with `ApproximateRate` I guess...

Reply:
It seems fine to me.

Reply:
yeah, I just got confused.  Thanks.